src/upload/demos/enUS/drag.demo.vue

```diff
@@ -9,6 +9,7 @@ You can set `directory-dnd` to `true` to make directory drag and drop available.
     multiple
     directory-dnd
     action="__HTTP__://www.mocky.io/v2/5e4bafc63100007100d8b70f"
+    :max="5"
```

Comment:
If this is necessary, please synchronously modify demo/zhCN/drag.demo.vue.

Reply:
Sorry, just added it... didn't want to create an extra example for it so I just took one of the existing ones to demonstrate the functionality 😉